modules/react/banner/lib/Banner.tsx

```diff
@@ -29,8 +30,9 @@ export interface BannerProps extends ExtractProps<typeof Flex, never> {
 const styles: CSSProperties = {
   cursor: 'pointer',
   transition: 'background-color 120ms',
+  outline: `${px2rem(1)} solid transparent`,
   '&:focus': {
```

Comment:
this changed status indicator to have a border?

<img width="1362" alt="image" src="https://github.com/user-attachments/assets/b1d45335-031f-49f0-9ead-e48861597430" />